### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by asingla1461** [anku94/imdb-rating-fetcher#2]
Display results sorted acc to rating ?

**Comment by anku94** [comment]
Results are already sorted according to similarity with the searched string. Adding a secondary sorting parameter doesn't make sense for lists of max len of 3.

**Comment by asingla1461** [comment]
@anku94  No, this is not what I am trying to say. I am talking about sorting all the movies in the folder according to the rating, Consider the case where the user wants to filter out the highly rated movies from the rest. They have to again manually figure out from the huge output.

Rating from each movie can be the maximum of 3 corresponding results.

If you still want to display the results according to original order, you can also use some sort of highlighting to render results which have a good rating, so the user can figure out.
